cluster/saltbase/salt/kube-apiserver/kube-apiserver.manifest

```diff
@@ -54,6 +54,10 @@
   {% set bind_address = "--bind-address=" + grains.publicAddressOverride -%}
 {% endif -%}
 
+{% set storage_backend = "" -%}
+{% if pillar['storage_backend'] is defined -%}
+  {% set storage_backend = "--storage-backend=" + pillar['storage_backend'] -%}
```

Comment:
Hmm - I'm wondering why it worked for me.

Anyway - waiting for you PR - I'm happy to review it.